vmhost/flags.go

```diff
@@ -3,6 +3,8 @@ package vmhost
 import "github.com/multiversx/mx-chain-core-go/core"
 
 const (
+	// AsyncV3Flag defines the flag that activates async v3
+	AsyncV3Flag core.EnableEpochFlag = "AsyncV3Flag"
```

Comment:
this has to be the last.

Reply:
also I made a cleanup somewhere of these flags, as they are not needed. Next week we need to make a merge from a new branch to feat/asyncV3